web-app/src/pages/tournament/TournamentDetails.vue

```diff
@@ -6,6 +6,14 @@
       class="q-px-xs-none q-px-sm-md"
       :class="{ fit: $q.screen.xs }"
     >
+      <q-tabs
+        v-if="tournament.type === 'round-robin'"
+        v-model="tab"
+        class="text-teal"
+      >
+        <q-tab label="Matches" name="matches" />
+        <q-tab label="Scoreboard" name="scoreboard" />
+      </q-tabs>
```

Comment:
What do you think about moving this tab bar below basic information about the tournament?

Reply:
I dont know if it is really necessary...

Reply:
My suggestion based on the fact that only the bottom of the page changes